vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -332,24 +309,16 @@ public Builder(String clientId, String clientSecret, String scope,
          * @param accountStore  (Optional, Recommended) An implementation that can be used to interface with Android's
          *                      <a href="https://goo.gl/QZ7rm">Account Manager</a>
          */
-        @Deprecated
-        public Builder(@Nullable String baseURLString,
-                       String clientId,
-                       String clientSecret,
-                       String scope,
+        public Builder(@NotNull String clientId,
```

Comment:
I think the base url should still be passed into the builder. This change would allow the client to create a builder without a base url. If the base url changes, the client should create a new configuration object and set it on the `VimeoClient` object.